Problem1_missing_number_in_sorted.py

```diff
@@ -0,0 +1,84 @@
+# APPROACH:1 BRUTE FORCE SOLUTION
+# Time Complexity : O(n^2) - n is the length of nums
```

Comment:
The Time Complexity will be O(n), adding and looking up in a hashset is a constant, O(1), operation.